apps/yggdrasil_web/web/controllers/character_controller.ex

```diff
@@ -64,4 +86,8 @@ defmodule YggdrasilWeb.CharacterController do
         render conn, :errors, data: err_changeset
     end
   end
+
+  defp admin?(conn) do
+    EnsurePermission.all?(conn, character: [:admin])
```

Comment:
If I understand this correctly, `:admin` is a permission. I don't know if `:admin` makes sense as a permission. Maybe more like `:view_all, :edit_all, :create_all`. I say that because `admin` is a role in the system to me not a permission necessarily.